src/App.js

```diff
@@ -1,18 +1,73 @@
 import React from 'react';
-import Component from './components/React.Component.jsx';
-import PureComponent from './components/React.PureComponent.jsx';
-import CreateElement from './components/React.CreateElement.jsx';
-import FComponent from './components/React.FunctionalComponent.jsx';
+import './assets/style/index.css';
+
+import Header from './components/Header/index.jsx';
+import Items from './components/Items/index.jsx';
+import Footer from './components/Footer/index.jsx';
+import ErrorBoundary from './components/ErrorBoundary.jsx';
+import items from './mock-data/data.json';
+
+import APP_STATES from './constants/APP_STATES';
+import SEARCH_BY from './constants/SEARCH_BY';
+import SORT_BY from './constants/SORT_BY';
+
+class App extends React.Component {
+    constructor() {
+        super();
+        this.state = {
```

Comment:
`state` can be directly in the component's body, it allows to get rid constructor.